qiskit/algorithms/gradients/base_qfi.py

```diff
@@ -29,12 +29,17 @@
 
 class BaseQFI(ABC):
     r"""Base class to computes the Quantum Fisher Information (QFI) given a pure,
-    parameterized quantum state. QFI is defined as:
+    parameterized quantum state. QFI is defined as [1]:
 
     .. math::
 
         \mathrm{QFI}_{kl}= 4 \mathrm{Re}[\langle \partial_k \psi | \partial_l \psi \rangle
             - \langle\partial_k \psi | \psi \rangle \langle\psi | \partial_l \psi \rangle].
+
+    **Reference:**
+    [1] Paris, Matteo G. A. (2009). Quantum estimation for quantum technology. International
```

Comment:
https://www.nature.com/articles/s41534-019-0187-2 and https://quantum-journal.org/papers/q-2020-05-25-269/pdf/ would probably be better references, as the one above doesn't use the same formulation and doesn't show the circuits 🙂